HIP/hip-869.md

```diff
@@ -101,141 +101,274 @@ Adopting a two-phase strategy, this approach facilitated the earlier release of
 This HIP proposes the introduction of a new NodeService API that enables a node operator to create, delete, and update nodes. All of these transactions must be signed by the Hedera Council.
 
 ```protobuf
-service NodeService {
-		/**
-     * Prepare to add a new node to the network.
-     * When a valid council member initiates a HAPI transaction to add a new node,
-     * then the network should acknowledge the transaction and update the network’s Address Book within 24 hours.   
-     * The added node will not be active until the network is upgraded.
+service AddressBookService {
+    /**
+     * A transaction to create a new consensus node in the network.
+     * address book.
+     * <p>
+     * This transaction, once complete, SHALL add a new consensus node to the
+     * network state.<br/>
+     * The new consensus node SHALL remain in state, but SHALL NOT participate
+     * in network consensus until the network updates the network configuration.
+     * <p>
+     * Hedera governing council authorization is REQUIRED for this transaction.
      */
-    rpc createNode (Transaction) returns (TransactionResponse);
+    rpc createNode (proto.Transaction) returns (proto.TransactionResponse);
 
     /**
-     * Prepare to delete the node to the network.
-	   * The deleted node will not be deleted until the network is upgraded.
-     * Such a deleted node can never be reused.
+     * A transaction to remove a consensus node from the network address
+     * book.
+     * <p>
+     * This transaction, once complete, SHALL remove the identified consensus
+     * node from the network state.
+     * <p>
+     * Hedera governing council authorization is REQUIRED for this transaction.
      */
-    rpc deleteNode (Transaction) returns (TransactionResponse);
+    rpc deleteNode (proto.Transaction) returns (proto.TransactionResponse);
 
     /**
-     * Prepare to update the node to the network.
-	   * The node will not be updated until the network is upgraded.
+     * A transaction to update an existing consensus node from the network
+     * address book.
+     * <p>
+     * This transaction, once complete, SHALL modify the identified consensus
+     * node state as requested.
+     * <p>
+     * This transaction MAY be authorized by either the node operator OR the
+     * Hedera governing council.
```

Comment:
This changed after we first wrote the text, it's only node operator (`admin_key`) now.
```suggestion
     * This transaction MUST be authorized by the node operator.
```

Reply:
Updated.